eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -129,6 +132,104 @@ void optimizesIfExpired(@TempDir final Path temp) throws Exception {
         );
     }
 
+    /**
+     * Test case for #1223.
+     *
+     * @param temp Temporary test directory.
+     * @throws Exception if unexpected error happened.
+     * @todo #1223:90min remove '@Disabled' annotation when caching of already optimized xmir files
```

Comment:
@volodya-lombrozo maybe it's worth to create puzzle for caching implementation?

Reply:
@mximp I've updated the puzzle description. Please, check.